ArcadiaTeamsBot.ServiceDesk.Abstractions/DTOs/RequestForCreationDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class RequestForCreationDTO
```

Comment:
Class name should not be the same as full sentence. It could better be renamed to `CreateRequestDTO`.

Reply:
Renamed